packages/api/src/modules/providers/apiAsso/apiAsso.service.test.ts

```diff
@@ -35,11 +36,19 @@ describe("ApiAssoService", () => {
     // @ts-expect-error: mock private method
     const findDocumentsMock = jest.spyOn(apiAssoService, "findDocuments") as jest.SpyInstance<any | null>;
 
+    let associationNameUpsert: jest.SpyInstance;
+
     const RNA = "W750000000";
 
+    beforeAll(() => {
+        // @ts-expect-error resolve value can must be an mongo updateObject, so just disable ts
```

Comment:
Value must be a mongo updateObject ? 

Mais si on a pas besoin du contexte je mettrai juste `// @ts-expect-error mock mongodb return value"`

Reply:
Done